src/Tizen.NUI/src/public/Theme/ExternalThemeManager.cs

```diff
@@ -151,9 +153,7 @@ private static void OnExternalPlatformThemeChanged(object sender, Tizen.Applicat
             id = e.Theme.Id;
             version = e.Theme.Version;
 
-            if (!ThemeManager.PlatformThemeEnabled) return;
```

Comment:
Just my curiosity :
Shouldn't we check 'Enabled' state anymore?
It looks like Enabled property could prevent the exception.

Reply:
Hi Seoyeon :)
You're right! It still need to check `PlatformThemeEnabled` to prevent exceptions.
I moved check code into a event handler.
Please check here :)
https://github.com/Samsung/TizenFX/pull/5080/files#diff-9c6352e4f8957996da4d6c32c6cb28f16418630ade639e3f42dff20e28cfed8cR600